CHANGELOG.md

```diff
@@ -5,7 +5,171 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## **8.0.0-alpha.11**
+## 8.0.0-alpha.12
+
+_Feb 14, 2025_
+
+We'd like to offer a big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve Polish (pl-PL) and Ukrainian (uk-UA) locales on the Data Grid
+- 🐞 Bugfixes
+
+Special thanks go out to the community contributors who have helped make this release possible:
+@mapache-salvaje, @Neonin, @nusr, @pawelkula, .
+Following are all team members who have contributed to this release:
+@alexfauquette, @arminmeh, @bernardobelchior, @cherniavskii, @Janpot, @JCQuintas, @KenanYusuf, @LukasTy, @MBilalShafi, @michelengelen, @oliviertassinari and @romgrk.
+
+### Data Grid
+
+#### Breaking changes
+
+- The `main--hasSkeletonLoadingOverlay` class has been renamed to `main--hiddenContent` and is now also applied when the "No columns" overlay is displayed.
+
+- The `apiRef.current.forceUpdate()` method was removed. Use selectors combined with `useGridSelector()` hook to react to changes in the state.
+
+- Fixes #16220
+
+  - Before: https://codesandbox.io/p/sandbox/intelligent-leavitt-q4h68n?from-embed=&workspaceId=ws_96AUJAP4MpFUq5gdSxRein
+  - After: https://codesandbox.io/p/sandbox/stupefied-scott-qnqpx5
+
```

Comment:
It looks like these lines do not belong here. 🙈 
```suggestion
```

Reply:
@MBilalShafi it you can rework your changelog breaking change to be more comprehensible :pray:

Reply:
Removed